test/CZMQToolkitTests.h

```diff
@@ -86,6 +86,7 @@ class CZMQToolkitTests : public /*::testing::Test*/ ZeroMQTests {
          zctx_destroy(&mContext);
       }
       
+      zctx_interrupted = false;
```

Comment:
Are you sure this is safe? You just raised SIGTERM which set it to true. Maybe for this one, it should go in the Setup function?